tests/test_bot_unlike.py

```diff
@@ -0,0 +1,155 @@
+
+from instabot.api.config import API_URL
+
+from .test_bot import TestBot
+from .test_variables import (TEST_CAPTION_ITEM, TEST_COMMENT_ITEM, TEST_PHOTO_ITEM,
+                             TEST_USERNAME_INFO_ITEM)
+
+
+class TestBotFilter(TestBot):
+
+    @responses.activate
+    @pytest.mark.parametrize('media_id,total,max_per_day', [
+        [111111, 1, 2],
+        [111111, 2, 2],
+        [111111, 3, 2],
+        ['111111', 1, 2],
+        ['111111', 2, 2],
+        ['111111', 3, 2]])
+    @patch('time.sleep', return_value=None)
+    def test_unlike(self, patched_time_sleep, media_id, total, max_per_day):
+        self.bot.total['unlikes'] = total
+        self.bot.max_per_day['unlikes'] = max_per_day
+        responses.add(
+            responses.POST, "{api_url}media/{media_id}/unlike/".format(
+                api_url=API_URL, media_id=media_id
+            ), json="{'status': 'ok'}", status=200
+        )
+        _r = self.bot.unlike(media_id)
+        test_r = _r if total < max_per_day else not _r
+        test_unliked = self.bot.total['unlikes'] == total + 1 if total < max_per_day else self.bot.total['unlikes'] == total
+        assert (test_r and test_unliked)
+
+    @responses.activate
+    @pytest.mark.parametrize('comment_id', [111111, '111111'])
+    @patch('time.sleep', return_value=None)
+    def test_unlike_comment(self, patched_time_sleep, comment_id):
+        responses.add(
+            responses.POST, "{api_url}media/{comment_id}/comment_unlike/".format(
+                api_url=API_URL, comment_id=comment_id
+            ), json="{'status': 'ok'}", status=200
+        )
+        _r = self.bot.unlike_comment(comment_id)
+        assert _r
+
+    @responses.activate
+    @pytest.mark.parametrize('media_ids,total,max_per_day', [
+        [[111111, 222222], 1, 3],
+        [[111111, 222222], 2, 3],
+        [[111111, 222222], 3, 3],
+        [['111111', '222222'], 1, 3],
+        [['111111', '222222'], 2, 3],
+        [['111111', '222222'], 3, 3]])
+    @patch('time.sleep', return_value=None)
+    def test_unlike_medias(self, patched_time_sleep, media_ids, total, max_per_day):
+        self.bot.total['unlikes'] = total
+        self.bot.max_per_day['unlikes'] = max_per_day
+        for media_id in media_ids:
+            responses.add(
+                responses.POST, "{api_url}media/{media_id}/unlike/".format(
+                    api_url=API_URL, media_id=media_id
+                ), json="{'status': 'ok'}", status=200
+            )
+        broken_items = self.bot.unlike_medias(media_ids)
+        test_unliked = self.bot.total['unlikes'] == max_per_day
+        test_broken = len(broken_items) == len(media_ids) - (max_per_day - total)
+        assert (test_unliked and test_broken)
+
+    @responses.activate
+    @patch('time.sleep', return_value=None)
+    def test_unlike_media_comments(self, patched_time_sleep):
+        my_test_comment_items = []
+        results = 5
+        for i, _ in enumerate(range(results)):
```

Comment:
I suggest this
```suggestion
        for i in range(results):
```
because you don't use elements